File: LV0/문자열 바꿔서 찾기.java
class Solution {

  public int solution(String myString, String pat) {
    String newMyString = "";
    for (int i = 0; i < myString.length(); i++) {
      if (myString.charAt(i) == 'A') {
        newMyString += 'B';
      } else if (myString.charAt(i) == 'B') {
        newMyString += 'A';
      }
    }
    return newMyString.contains(pat) ? 1 : 0;
  }
}

File: LV0/문자열 바꿔서 찾기.py
def solution(myString, pat):
    newMyString = ""
    for i in range(len(myString)):
        if myString[i] == 'A':
            newMyString += 'B'
        elif myString[i] == 'B':
            newMyString += 'A'
    return 1 if pat in newMyString else 0